generated/nidcpower/nidcpower/session.py

```diff
@@ -6911,11 +6911,11 @@ def __init__(self, resource_name, channels=None, reset=False, options={}, indepe
         options = _converters.convert_init_with_options_dictionary(options)
 
         # Call specified init function
-        # Note that _library_interpreter sets _vi to 0 in its constructor, so that if
+        # Note that _interpreter clears the session handle in its constructor, so that if
```

Comment:
I think this part needs some more wordsmithing. How do you clear something before it's been set? And how do you reference a value that you've cleared? Maybe it would be better to say default-initializes? @marcoskirsch may be able to help us out when he returns. I don't think we need to hold up this PR for it.

Reply:
@DavidCurtiss can handle this is a separate PR. I'm going to merge.